src/opensles/AudioStreamOpenSLES.cpp

```diff
@@ -13,7 +13,7 @@
  * limitations under the License.
  */
 #include <sys/types.h>
-#include <cassert>
+#include <assert.h>
```

Comment:
Just a question: since we're doing `NULL` -> `nullptr` which is C to C++, why are we then doing `<cassert>` to `<assert.h>` which is C++ to C header?

Reply:
The idea was just to get all headers uniform, following a comment in #352. You are right, I will convert <assert.h> into <cassert> instead

Reply:
*into <cassert>